tests/htcanalyze_test.py

```diff
@@ -245,7 +232,7 @@ def test_log_to_dict(htcan):
         'Execution details': ['Process was', 'Submitted from', 'Executing on'],
         'Values': ['[red]Aborted[/red]', '10.0.8.10', '10.0.9.1']}
 
-    assert res_dict == {}
+    assert res_dict == []
```

Comment:
It's called `res_dict`, but you expect it to be an empty list? Why call it `res_dict` then? This is kind of the issue with putting the variable type in its name. All of a sudden you've lost all of the flexibility of changing its type without causing confusion.

Reply:
Thanks for mentioning, I will stay away from this kind of definitions. 👍